src/components/shared/__tests__/Validations.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import 'jest-enzyme';
+
+import { RegularNode } from '@stoplight/json-schema-tree';
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import { Validations } from '../../shared';
+import { getValidationsFromSchema } from '../Validations';
+
+describe('Validations component', () => {
+  it('should render number type validations', () => {
+    const node = new RegularNode({
+      type: 'number',
+      minimum: 10,
+      maximum: 40,
+      multipleOf: 10,
+      default: 20,
+      example: 20,
+      enum: [10, 20, 30, 40],
+    });
+
+    const validations = getValidationsFromSchema(node);
+    const wrapper = mount(<Validations validations={validations} />);
+
+    expect(wrapper).toIncludeText('>= 10<= 40');
```

Comment:
You could probably just do
```suggestion
    expect(wrapper.text()).toMatchInlineSnapshot(
```
and verify the output

Reply:
Do we like snaphosts? :wink:

Reply:
😭

Reply:
Well, I don't like them, BUT inline snapshots are okay.